functions/check-started/index.js

```diff
@@ -28,17 +28,17 @@ module.exports = async function (context, checkStartMessage) {
   }
 
   // Update the admin database to update the check status to Check Started
-  try {
-    await updateAdminDatabaseForCheckStarted(
-      checkStartMessage.checkCode,
-      new Date(checkStartMessage.clientCheckStartedAt),
-      context.log
-    )
-    context.log(`check-started: SUCCESS: Admin DB updated for ${checkStartMessage.checkCode}`)
-  } catch (error) {
-    context.log.error(`check-started: ERROR: unable to update admin db for ${checkStartMessage.checkCode}`)
-    throw error
-  }
+  // try {

```

Comment:
can we remove these?

Reply:
The entire functions directory will be removed very shortly in a subsequent PR.  I say we leave it as is for now, as it is clear that this used to be functional.